code/modules/projectiles/guns/ballistic/automatic.dm

```diff
@@ -564,16 +564,18 @@
 	suppressor_y_offset = 28
 
 
-//Police rifle			Keywords: OASIS, 5.56mm, Semi-auto, 30 (10-50) round magazine
+//Police rifle			Keywords: OASIS, 5.56mm, Semi-auto, 20 (10-50) round magazine
 /obj/item/gun/ballistic/automatic/marksman/policerifle
 	name = "Police  rifle"
-	desc = "This old pre-war rifle made for law enforcement has been passed down generations in the Oasis Police Department, carefully repaired and rebuilt whenever damaged. Dried and crumbling polymers have been replaced with wood and reinforcing strips of leather, the black finish worn down a century ago, but it still works, as good as ever."
+	desc = "A pre-war Rifle that has been constantly repaired and rebuilt by the Oasis Police Department. Held together by duct tape and prayers, it somehow still shoots."
 	icon = 'icons/fallout/objects/guns/ballistic.dmi'
 	lefthand_file = 'icons/fallout/onmob/weapons/guns_lefthand.dmi'
 	righthand_file = 'icons/fallout/onmob/weapons/guns_righthand.dmi'
 	icon_state = "rifle-police"
 	item_state = "assault_carbine"
-	init_mag_type = /obj/item/ammo_box/magazine/m556/rifle/extended
+	init_mag_type = /obj/item/ammo_box/magazine/m556/rifle
+	extra_damage = -1
+	spread = 1.1
```

Comment:
I'll consider this a balance change, what exactly was the reason behind it?

Reply:
its the worn-down modifier thats supposed to be applied to weapons that are worn-down, since i was under the impression the marksman carbine, the parent weapon of the police rifle, was too high a tier